### PR TITLE
V3 remove six

### PR DESCRIPTION
**Pull request by rekby**
I hereby agree to the terms of the CLA available at: https://yandex.ru/legal/cla/?lang=en

## Pull request type

Please check the type of change your PR introduces:

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [x] Other (please describe):

## What is the current behavior?
Remove six package - v3 doesn't support python2

### CODE REVIEW

File: ydb/aio/credentials.py [hunk 1]
```diff
@@ -55,8 +54,9 @@ def submit(self, callback):
             asyncio.ensure_future(self._wrapped_execution(callback))
 
 
-@six.add_metaclass(abc.ABCMeta)
-class AbstractExpiringTokenCredentials(credentials.AbstractExpiringTokenCredentials):
+class AbstractExpiringTokenCredentials(
+    credentials.AbstractExpiringTokenCredentials, abc.ABC
```
Comment (Valeria1235) [inline ydb/aio/credentials.py:58]: i think there is no need to inherit from abc.ABC becouse we have already inherited from abc-class.

File: ydb/aio/iam.py [hunk 2]
```diff
@@ -24,8 +23,7 @@
     aiohttp = None
 
 
-@six.add_metaclass(abc.ABCMeta)
-class TokenServiceCredentials(AbstractExpiringTokenCredentials):
+class TokenServiceCredentials(AbstractExpiringTokenCredentials, abc.ABC):
```
Comment (Valeria1235) [inline ydb/aio/iam.py:26]: same as above

File: ydb/types.py [hunk 3]
```diff
@@ -12,12 +11,8 @@
 
 _SECONDS_IN_DAY = 60 * 60 * 24
 _EPOCH = datetime(1970, 1, 1)
-if six.PY3:
-    _from_bytes = None
-else:
 
-    def _from_bytes(x, table_client_settings):
-        return _utilities.from_bytes(x)
+_from_bytes = None
```
Comment (Valeria1235) [inline ydb/types.py:15]: it would be nice to remove `_from_bytes` at all

Comment (rekby) [reply ydb/types.py]: great!

**Review by Valeria1235 — APPROVED** [review]

**Pull request merged**